assets/src/scss/layout/_footer.scss

```diff
@@ -60,15 +62,11 @@
     padding: 55px 0 $n25;
   }
 
-  p {
-    font-family: $roboto;
-  }
-
-  a {
-    color: var(--footer_links_color, inherit);
+  a --footer--links-- {
```

Comment:
Note that there is both this one which targets any `a` tag in the footer, but there's also the `.footer-links` class, which is a menu. Could be confusing, maybe we need to rename one of the two?

Reply:
Good point, maybe the class could be renamed `.footer-menu`?

Reply:
Makes sense :ok_hand: Probably better to rename the class separately, but we can already give it the right name instead of generating it from the class (just pushed this change) https://github.com/greenpeace/planet4-master-theme/pull/1325/files#diff-8b7c0326c01b36e050f9be000f365759c8fd1ba4653b49ddd09eec4bba3f3f11R152